glide.lock

```diff
@@ -4,7 +4,7 @@ imports:
 - name: github.com/boltdb/bolt
   version: 831b652a7f8dbefaf94da0eb66abd46c0c4bcf23
 - name: github.com/btcsuite/btcd
-  version: bd4e64d1d43bad445dd8e6577907c0c265cd83c2
+  version: a7b35d9f9e24f8cebf3dc30cf083668700690095
```

Comment:
Yeah, `glide up` tries to update all kinds of other stuff (adding entries that don't belong), so I just did it manually.